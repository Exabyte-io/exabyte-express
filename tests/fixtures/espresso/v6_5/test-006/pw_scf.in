version https://git-lfs.github.com/spec/v1
oid sha256:a375f4ebadbd23f7b25d8da06d4efd0aa97b5e0cf40e260d7bc305ed61cac28e
size 1070
