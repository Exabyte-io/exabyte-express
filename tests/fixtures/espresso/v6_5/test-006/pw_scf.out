version https://git-lfs.github.com/spec/v1
oid sha256:9d3b441a251d16fe672df82f85e73c785777f1f79e155a846ac3138362339d37
size 11493
